### ISSUE TITLE
ITC-2 sends invalid alarms

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_implementations#8]
The ITC-2 simulator sends invalid alarms:

```
1) RSMP site alarm A0302 detector error (logic error)
     Failure/Error: raise SchemaError.new error_string

     RSMP::SchemaError:
       invalid Alarm, schema errors: /aSp enum, /aS enum, /cat enum {"mType":"rSMsg","type":"Alarm","mId":"2b701db6-7624-43b8-889b-03676190edb4","ntsOId":"KK+AG9998=001TC000","xNId":"","cId":"KK+AG9998=001DL036","aCId":"A0301","xACId":"C_DET_OFF (36, 36, 0) : Detector #36","xNACId":"","aSp":"issue","ack":"notAcknowledged","aS":"active","sS":"notSuspended","aTs":"2021-02-10T02:07:20.000Z","cat":"b","pri":"3","rvs":[{"n":"detector","v":"36"},{"n":"type","v":"loop"},{"n":"errormode","v":"off"},{"n":"manual","v":"False"}]}
```

aSp must be one of [ "Issue", "Acknowledge", "Suspend", "Resume" ]
aS must be one of [ "inActive", "Active" ]

(note the capitalization in the above)

cat must be one of [ "T", "D" ]

The cat issue was fixed in versions 6.96.0 of the ITC-3, see #4. What about the other two?

**Comment by emiltin** [comment]
How strict is the spec in relation to capitalization?

I do recommend we are strict and enforce a specific capitalization, since it will reduce the chance of errors, and will make implementations simpler. For example, there is no case-insensitively enums in json schemaa.

But if it wasn't a requirement in previous version of core, we might accept different capitalizations in the validator when running older versions of core. However this would be a head-ache to implement, since there is no way to do case-insensitive enum checks.

**Comment by otterdahl** [comment]
We had an old question in the [faq](https://github.com/rsmp-nordic/rsmp_core/blob/master/faq.md#4) regarding this. It is not specified as required, but the recommendation is to treat RSMP as case sensitive. I think it is a good idea to add this as a requirement in future versions.

**Comment by emiltin** [comment]
Yes and there's ann issue here: https://github.com/rsmp-nordic/rsmp_core/issues/35.
But the question is how we should validate existing and older versions. Do we want we accept arbitrary casing? Unfortunataly there's no way to do that with json schema. Or can be get by by allow e.g. either "Issue" or "issue"?
